SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -186,19 +229,46 @@ QueryConstraint InterestFactory::CreateSystemDefinedConstraints()
 	return SystemDefinedConstraints;
 }
 
-QueryConstraint InterestFactory::CreateUserDefinedConstraints()
+QueryConstraint InterestFactory::CreateCheckoutRadiusConstraints()
 {
-	return QueryConstraint{};
-}
+	// Checkout Radius constraints are defined by the ClientInterestDistance property on actors.
+	//   - Checkout radius is a RelativeCylinder constraint on the player controller.
+	//   - ClientInterestDistance on AActor is used to define the default checkout radius with no other constraints.
+	//   - ClientInterestDistance on other actor types is used to define additional constraints if needed.
+	//   - If a subtype defines a radius smaller than a parent type, then its requirements are already captured.
```

Comment:
If a subtype has a smaller radius then the parent, then don't we want to enforce that it's not checked out at the greater checkout radius? As in the smaller radius type overrides the parent's for those subtypes?

Reply:
Okay to do this in another PR, but could you create a tracking ticket?

Reply:
https://improbableio.atlassian.net/browse/UNR-1622